code/_onclick/ai.dm

```diff
@@ -34,6 +34,12 @@
 	if(control_disabled || stat)
 		return
 
+	for(var/C in eyeobj.visibleCameraChunks)
```

Comment:
replace all of this with:

```
if(!checkTurfVis(get_turf_pixel(A))
    return
```

this will allow the AI to still click on things we normally allow them to (pixel shifted objects, for example)